test/integration/pack/QtiItemPackerTest.php

```diff
@@ -86,6 +86,29 @@ public function testPackingInvalidQtiItem()
         $itemPackerMock->packItem(new core_kernel_classes_Resource('foo'), 'en-US', $this->getDirectoryStorage());
     }
 
+    /**
+     * Test packing an item where QTI content isn't valid, but validation skipped
+     */
+    public function testPackingInvalidQtiItemWithoutValidation()
+    {
+        $filename = dirname(__FILE__) . '/../samples/wrong/notvalid_associate.xml';
+        $this->assertTrue(file_exists($filename));
+
+        $itemPackerMock = $this
+            ->getMockBuilder('oat\taoQtiItem\model\pack\QtiItemPacker')
+            ->setMethods(['getXmlByItem'])
+            ->getMock();
+
+        $itemPackerMock
+            ->method('getXmlByItem')
+            ->will($this->returnValue(file_get_contents($filename)));
+        $itemPackerMock->setSkipValidation(true);
+
+        $itemPackerMock->packItem(new core_kernel_classes_Resource('foo'), 'en-US', $this->getDirectoryStorage());
+
+        $this->assertTrue(true);
```

Comment:
This is useless

Reply:
Why not?
As you see, the previous test allow us to throw an exception, if the item is not valid.
In current test the same situation, but we need to check the attribute `skipValidation` allow us to skip this validation (do not throw exception). So, if `packItem` will not throw any exception - execution will not stop and we will go to the last assert.

If you have any ideas how to test it better - I will be happy to change this part of code 😄